zulipterminal/ui_tools/buttons.py

```diff
@@ -561,13 +574,29 @@ def _switch_narrow_to(self, parsed_link: ParsedNarrowLink) -> None:
                 topic_name=parsed_link["topic_name"],
                 contextual_message_id=parsed_link["message_id"],
             )
+        elif "pm-with:near" == narrow:
+            emails = list(
```

Comment:
It would be cleaner to only keep the narrowing part in the method like other conditionals.